src/Compute.js

```diff
@@ -0,0 +1,210 @@
+    rawAlgoMeta.rawcode = textRawAlgo
+    return startCompute(undefined, rawAlgoMeta)
+  }
+
+  async function getStatus() {
+    try {
+      const accounts = await ocean.accounts.list()
+
+      const status = await ocean.compute.status(accounts[0], agreementId, jobId)
+      setJobStatus(JSON.stringify(status, null, '\t'))
+      console.log(status)
+    } catch (error) {
+      console.error(error.message)
+    }
+  }
+
+  async function showDivAlgo() {
+    setIsAlgoInputVisible(isAlgoInputVisible ? false : true)
+  }
+
+  async function updateRawAlgoCode(event) {
+    setTextRawAlgo(event.target.value)
+  }
+
+  async function updateDdoAssetId(event) {
+    setDdoAssetId(event.target.value)
+  }
+  async function handlePublishOutputState(event) {
+    setPublishOutputState(event.target.checked)
+  }
+  async function handlePublishLogState(event) {
+    setPublishLogState(event.target.checked)
+  }
+
+  if (!web3) {
+    return null
+  }
+
+  return (
+    <>
+      <h3>Compute</h3>
+      <ComputeSection>
+        <button onClick={publish}>Publish dataset with compute service</button>
+        Asset DID:
+        <input type="text" value={ddoAssetId} onChange={updateDdoAssetId} />
+      </ComputeSection>
+      <ComputeSection>
+        <button onClick={publishalgo}>Publish algorithm</button>
+        Algo DID:
+        <input type="text" value={ddoAlgorithmId} readOnly />
+      </ComputeSection>
+      <ComputeSection>
+        <input
+          type="checkbox"
+          checked={publishOutputState}
+          onChange={handlePublishOutputState}
+        />
+        Publish Output into the Marketplace
+        <input
+          type="checkbox"
+          checked={publishLogState}
+          onChange={handlePublishLogState}
+        />
+        Publish Algorithm Logs into the Marketplace
+        <button
+          onClick={startWithPublishedAlgo}
+          disabled={!ddoAssetId || !ddoAlgorithmId}
+        >
+          Order and start compute service with published algorithm
+        </button>
+        <button onClick={startWithRawAlgo} disabled={!ddoAssetId}>
+          Order and start compute service with raw algorithm
+        </button>
+        <button onClick={showDivAlgo}>Show/Hide Raw Algo</button>
+        <div style={{ display: isAlgoInputVisible ? 'block' : 'none' }}>
+          <textarea
+            rows="10"
+            cols="120"
+            value={textRawAlgo}
+            onChange={updateRawAlgoCode}
+          />
+        </div>
+        Job ID:
+        <input type="text" value={jobId} readOnly />
+      </ComputeSection>
+      <ComputeSection>
+        <button onClick={getStatus} disabled={!jobId}>
+          Get Job Status
+        </button>
+        Compute status:
+        <textarea rows="15" cols="120" value={jobStatus} readOnly />
+      </ComputeSection>
+    </>
+  )
+}
+
+function ComputeSection({ children }) {
+  return (
+    <>
+      <div
+        style={{
+          display: 'flex',
```

Comment:
doing `text-align: center` is probably all you need for what you want to achieve. Which is already on the whole page. Then work with breaking elements like `<p>` to create line breaks in your sections. Then this whole thing here becomes redundant and we don't add so much confusing markup